utils/deploy/envoy.yaml

```diff
@@ -212,7 +204,7 @@ data:
                                           "scope": "rlp-ns-B/rlp-name-B",
                                           "data": [
                                             {
-                                              "static": {
+                                              "expression": {
                                                 "key": "rlp-ns-B/rlp-name-B/limit-not-to-be-activated",
                                                 "value": "1"
```

Comment:
While this would work, this also changed the type tho! It's now `int(1)` ... but will be erased and will be `1`, just as if you had it as `"\"1\""` in there, which would be closer semantically to what this was before...

tl;dr _not wrong_, but... ;)